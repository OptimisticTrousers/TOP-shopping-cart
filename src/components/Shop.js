export default function Shop() {
    return (
        <main>
            <h1>Get Clothing!</h1>
            <p>Times are tough. Liven up with our top-notch clothing!</p>
            <div className="products">

            </div>
        </main>
    )
}