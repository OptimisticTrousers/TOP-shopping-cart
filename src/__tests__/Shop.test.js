import React from 'react'
import {render, screen} from '@testing-library/react'
import '@testing-library/jest-dom'
import userEvent from '@testing-library/user-event'
import Shop from '../components/Shop';

it("renders navbar", () => {

    const {asFragment} = render(<Shop/>)

    expect(asFragment()).toMatchSnapshot()
})