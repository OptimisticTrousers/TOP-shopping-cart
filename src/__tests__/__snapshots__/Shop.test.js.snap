// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop renders navbar 1`] = `
<DocumentFragment>
  <div
    class="shop"
  >
    <h2
      class="shop-title"
    >
      Get Clothing!
    </h2>
    <p
      class="shop-description"
    >
      Times are tough. Liven up with our top-notch clothing!
    </p>
    <div
      class="products"
    />
  </div>
</DocumentFragment>
`;

exports[`renders navbar 1`] = `
<DocumentFragment>
  <div
    class="shop"
  >
    <h2
      class="shop-title"
    >
      Get Clothing!
    </h2>
    <p
      class="shop-description"
    >
      Times are tough. Liven up with our top-notch clothing!
    </p>
    <div
      class="products"
    />
  </div>
</DocumentFragment>
`;
